### PR TITLE
LPS-116827 Add Resource Command URL to DropdownItem

### PR DESCRIPTION
**Pull request by cgoncas**
Hey @carloslancha !!

I have sent this pull directly to you since it is a work in progress.

Let me know if you can continue from here.

Thanks!

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:tesf:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java [hunk 1]
```diff
@@ -95,6 +99,28 @@ public ContentDashboardDropdownItemsProvider(
 			() -> {
 				DropdownItem dropdownItem = new DropdownItem();
 
+				dropdownItem.setData(
+					HashMapBuilder.<String, Object>put(
```
Comment (carloslancha) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java:103]: We can add here the `action` property with `showInfo` as value, and remove it from https://github.com/carloslancha/liferay-portal/pull/275/files#diff-82a96a032d82cb080686b01630ae5968R122

**Comment by carloslancha** [comment]
@cgoncas apart from https://github.com/carloslancha/liferay-portal/pull/275#discussion_r458805378, LGTM. Do you want me to include it in my pr for https://issues.liferay.com/browse/LPS-116826 ?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java (hunk 1 above)
Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java:103]: Awesome!!

**Comment by cgoncas** [comment]
Hey @carloslancha !!

I will add the changes that you have mentioned and will use the https://issues.liferay.com/browse/LPS-116826. Thanks!!

**Comment by cgoncas** [comment]
Hey @carloslancha !!

I have updated this branch, so feel free to include these changes in https://issues.liferay.com/browse/LPS-116826.

Thanks!

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
@cgoncas ok! I'm adding'em to https://github.com/liferay-frontend/liferay-portal/pull/152 then 😄 Thx!

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
-> https://github.com/liferay-frontend/liferay-portal/pull/152

**Comment by carloslancha** [comment]
ci:stop

**Pull request closed without merging**